### PR TITLE
Update server readme

### PR DESCRIPTION
**Pull request by Saneeitas**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+Clone the repository:
+
+```bash 
+git clone https://github.com/pi-apps/pailot-for-pi.git
+
+```
+
+Navigate to the project directory:
+
+```TypeScript
```
Comment (vyktoremario) [inline server/README.md]: This should not be typescript. It is not a typescript code.
It is a shell or bash command.

File: server/README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+cd server
+
+```
+
+Install the dependencies:
+
+```TypeScript
```
Comment (vyktoremario) [inline server/README.md]: Same as above

File: server/README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+npm install OR yarn install
+
+```
+
+# Compiling the TypeScript code
+
+To compile the TypeScript code, run the following command in the terminal:
+
+```TypeScript
```
Comment (vyktoremario) [inline server/README.md]: Same as above

File: server/README.md [hunk 4]
```diff
@@ -1 +1,67 @@
...
+npm run build OR yarn build
+
+```
+
+# Running the application 
+
+To start the application, run the following command in the terminal:
+
+```TypeScript
```
Comment (vyktoremario) [inline server/README.md]: Same as above

File: server/README.md [hunk 5]
```diff
@@ -1 +1,67 @@
...
+npm start OR yarn start
+
+```
+
+The application should now be running at:
+```
+ "http://localhost:3333"
```
Comment (vyktoremario) [inline server/README.md:59]: This is not supposed to be in a fence.

**Review by vyktoremario — CHANGES_REQUESTED** [review]
I left some comments.
Please fix the comment in this PR. Stop closing PR to fix comments

File: server/README.md (hunk 5 above)
Comment (Saneeitas) [reply server/README.md:59]: Okay I Will do that

**Review by vyktoremario — APPROVED** [review]

**Pull request merged**